Packs/Wiz/.pack-ignore

```diff
@@ -0,0 +1,2 @@
+[file:Wiz.yml]
+ignore=IN145
```

Comment:
ACK. Done. Those are leftovers, so cleaned it up.

As for credentials, since it wasn't mandatory, we didn't add it.
I'll look at the LOE to add it to this PR.

Reply:
917b05ee911e1bce12d1b0f8f43661e89a99b596

Reply:
I also switched to credentials here: bb4954bfe8e32d1b967bcd12c0974575efa3eee3